tests/monster_test.grpc.fb.py

```diff
@@ -0,0 +1,106 @@
+import grpc
```

Comment:
All Python files are in the namespaced directory `Mygame\Example`, so should this file.